lib/key-handle-decorator.js

```diff
@@ -0,0 +1,64 @@
+/* @flow */
+
+import React from 'react';
+
+import KeyHandler from './key-handler';
+import {type KeyConstants} from './constants';
+import {matchesKeyboardEvent, eventKey} from './utils';
+import { type rawOrArray } from './types';
+
+export type KeyhandleDecoratorState = {|
+ keyValue?: ?rawOrArray<string>,
```

Comment:
Wonder if there's a better name for `raw`, or is this something commonly used?

Reply:
I like `primitiveOrArary`

Reply:
Fixed